fjord/l10n/utils.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+
+_COLLAPSE_RE = re.compile(r'\s+', re.UNICODE)
+
+
+def collapse_whitespace(message):
+    """Collapses consecutive whitespace into a single space"""
+    return _COLLAPSE_RE.sub(' ', message).strip()
```

Comment:
Out of curiosity, how well does that work in the single line and (say) 10-line cases?